runbooks/english/06-use-interactive-dagger-shell.md

```diff
@@ -0,0 +1,149 @@
+# Introducing Dagger Shell
+
+The Mar 26, 2025, Dagger.io's team [announce](dagghttps://dagger.io/blog/a-shell-for-the-container-age-introducing-dagger-shell) an amazing feature: Dagger Interractive Shell.
+
+This feature is very usefull to test Dagger.io and allow Dagger to be used by non-developper people.
+
+Dagger shell is not a shell like Bash, Zsh...It is more like [jq](https://jqlang.org) syntax.
+
+Let's go started.
+
+## Launch interactive shell
+
+To start Dagger shell, just type:
+```
+❯ dagger
+Dagger interactive shell. Type ".help" for more information. Press Ctrl+D to exit.
+⋈
+─ esc nav mode · > run prompt ────────────────────────────────────────────────────
+```
+
+You can exit Dagger shell by using `Ctrl+C`, `Ctrl+D` or `exit`.
+
+# Get help
+
+In Dagger shell, you can use `.help` command to display help:
+```
+✔ .help 0.0s
+  cache-volume   Constructs a cache volume for a given cache key.                                                                                                                                                                           
+  container      Creates a scratch container.                                                                                                                                                                                               
+  directory      Creates an empty directory.
+...
+```
+
+You can have information on module or command. See [Built-in help](https://docs.dagger.io/features/shell/#built-in-help) section of official documentation.
+
+# Run first command
+
+Now, you can enter any Dagger command.
+
+Enter this command:
+```
+container | from alpine | with-exec whoami
+```
+
+You tell to Dagger to run a container based on alpine Docker image and run commande `whoami`. But wait, you have this output:
+```
+✔ container | from alpine | with-exec whoami 0.0s
+Container@xxh3:83a64a1566b1bdba
+```
+
+You don't see the result of `whoami` command!
```

Comment:
how to have the result of whoami command. It's strange to change the example with echo after